label_studio/projects/api.py

```diff
@@ -589,14 +597,16 @@ def filter_queryset(self, queryset):
 
     def delete(self, request, *args, **kwargs):
         project = generics.get_object_or_404(Project.objects.for_user(self.request.user), pk=self.kwargs['pk'])
+        task_ids = list(Task.objects.filter(project=project).values('id'))
         Task.objects.filter(project=project).delete()
-        return Response(status=204)
+        emit_webhooks_for_instance(request.user.active_organization, None, WebhookAction.TASKS_DELETED, task_ids)
+        return Response(data={'tasks': task_ids}, status=204)
 
 
     def get(self, *args, **kwargs):
         return super(TasksListAPI, self).get(*args, **kwargs)
 
-    @swagger_auto_schema(auto_schema=None)
```

Comment:
I'm worried removing this will cause this POST to unintentionally appear in the API docs...

Reply:
I turned it back.